eng/Packages.Data.props

```diff
@@ -388,6 +388,7 @@
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.2" />
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.8.0" />
     <PackageReference Update="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Update="Microsoft.CodeAnalysis.CSharp" Version ="4.12.0" PrivateAssets="All" />
```

Comment:
we should consider an override in the one csproj that wants to differ from the other entry in this file.